pallets/capacity/src/types.rs

```diff
@@ -55,6 +55,7 @@ impl<T: Config> StakingAccountDetails<T> {
 	}
 
 	#[cfg(any(feature = "runtime-benchmarks", test))]
+	#[allow(clippy::unwrap_used)]
```

Comment:
As this in the runtime, should we be allowing it?  Is there a way to display a warning but allow the build to proceed?  e.g.  we don't have to drop everything and fix it but be aware that we should fix it at some point?

Reply:
The comment for the function says it's for testing only and it has `feature = "runtime_benchmarks"` or `feature = test`, so I thought it was ok.
`make lint` promotes `warnings` to errors, so I'm not sure if we can adjust that.